### PR TITLE
Added docs for command line auth provider

### PR DESCRIPTION
**Pull request by bob1de**
**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19985

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by bob1de** [comment]
@klaasnicolaas Sorry for dismissing the review, needed to unify line breaks and added an external link, but I don't know whether that's against any project rule or not.

**Comment by klaasnicolaas** [comment]
I think that it is no problem to add an external link. 

After that:
✅ PR can be merged as soon as the parent PR gets merged.

**Review by balloob — APPROVED** [review]

**Pull request merged**
